### DISCUSSION TITLE
How to separate a whole region into subregions with gcambreakout

### DISCUSSION THREAD

**Discussion by nihongi** [JGCRI/gcam-core#488] [Q&A]
I'm considering developing GCAM-Japan. To do this, we looked at the code for GCAM-China, GCAM-Korea and so on, and understood that they had plugged in a number of chunks to gcamdata.
I think one way would be to follow their method, but on the other hand, I'm thinking that we could use gcambreakout instead and do the same thing.
When I look at the user guide for gcambreakout, it shows how to separate a specific country from an existing region. If I am to create GCAM-Japan, I would be splitting the Japan region into 47 prefectures. Is it possible to use gcambreakout in this way? Is there any precedent for this?

**Comment by pkyle** [comment]
Unfortunately, `gcambreakout` will not help with this. That is only used for breaking countries out of multi country aggregate regions in the model (e.g. splitting Thailand out of Southeast Asia); it is based on the principle that all of the input data used in `gcamdata` is either at the country level (or finer resolution in the case of gridded data), or at least can be downscaled to the country level in the data processing. Splitting a nation into its own political sub-regions is an entirely different process, and in our experience so far, it's different in every country due to the nature of the sub-national data that are available. That's good that you have access to three different methods by which this has been done; I guess I would recommend taking a look at each of them and seeing which country’s data is most similar to the sub-national data from Japan that you have access to, and which approach structurally yields the sort of outcome you're aiming for. If you decide to use GCAM-USA as a template, I would recommend not following that same approach for the electric power sector, and instead of just leaving one national grid where all subregions will see the same price, without the load duration segments (e.g.baseload vs peak).
